app/src/renderer/vuex/modules/distribution.js

```diff
@@ -0,0 +1,178 @@
+import * as Sentry from "@sentry/browser"
+import Vue from "vue"
+import { coinsToObject } from "scripts/common.js"
+
+export default ({ node }) => {
+    const emptyState = {
+        loading: false,
+        loaded: false,
+        error: null,
+        /* totalRewards use the following format:
```

Comment:
i don't know if i like having these comments like this.

Reply:
if we don't have types this is a good documentation on the formats. what is bugging you with them?

Reply:
they take up many lines and we don't do this anywhere else.

Reply:
I think here the problem we have is `we don't do this anywhere else`, it's true, but we should